stable/fluentd-shipper/templates/daemonset.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ .Chart.Name }}
+  labels:
+    app: {{ .Chart.Name }}
+    chart: {{ .Chart.Name }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  updateStrategy:
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      app: {{ .Chart.Name }}
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Chart.Name }}
+        release: {{ .Release.Name }}
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/configmaps.yaml") . |  sha256sum  }}
+    spec:
+      tolerations:
+{{ toYaml .Values.tolerations | indent 8 }}
+      imagePullSecrets:
+      - name: dockercfg
```

Comment:
I think this needs to be in values so we can change what secret to use

Reply:
will be implemented in INFRA-8676